DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -1979,6 +2110,341 @@ static function MarkBingo(DXRando dxr, coerce string eventname)
     }
 }
 
+static simulated function string GetBingoGoalHelpText(string event,int mission)
```

Comment:
should this function just be in BingoTile.uc? I feel like DXREvents is already pretty big lol

or PersonaScreenBingo, or even in BingoHintMsgBox